code/_onclick/click.dm

```diff
@@ -253,8 +256,21 @@
 	A.CtrlShiftClick(src)
 	return
 
-/atom/proc/CtrlShiftClick(var/mob/user)
+/atom/proc/CtrlShiftClick(var/mob/living/user) 	//PybroPointClick, 1/30/15
+	if(istype(user, /mob/living/carbon/brain || /mob/living/silicon/pai)) //If this isn't in place, then they can still point. They won't actually make an arrow/emote, but they'll still get the tired finger message. Pedantic, but whatever.
+		return
+	if(user.client && user.client.eye == user)	//Makes sure all of our seeing is in order.
+		if(user.pointcooldown == 1)				//This is to prevent spam.
+			user << "<span class='notice'>Give your finger a moment to cool down!</span>" //Cheesy, but whatever.

```

Comment:
That's the issue, it's difficult to come up with one. if no better replacement comes along, this one can stay. or some variant on "your arm is resting" or such.